.github/actions/backup-and-restore-snapshot-database/action.yml

```diff
@@ -28,6 +28,13 @@ runs:
   using: composite
 
   steps:
+    - name: Set Environment variables
+      id: set_env_var
+      shell: bash
+      run: |
+        tf_vars_file=terraform/application/workspace_variables/${{ inputs.environment }}.tfvars.json
+        echo "namespace=$(jq -r '.namespace' ${tf_vars_file})" >> $GITHUB_ENV
```

Comment:
We normaly follow a convention to use uppercase for environment variable names. Can you rename it in the workflows?